tools/pom.xml

```diff
@@ -138,6 +142,90 @@
                 <groupId>net.alchim31.maven</groupId>
                 <artifactId>scala-maven-plugin</artifactId>
             </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
we should include the LICENSE file in here as well: https://github.com/NVIDIA/spark-rapids/blob/branch-22.06/sql-plugin/pom.xml#L368

Reply:
Done!